src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -1061,3 +1022,86 @@ class TestLibrary(SystemTests):
+
+    # Test doesn't run over gRPC due to issues with the name of the property reference_value.
+    @pytest.mark.resource_name("4190/0")
+    @pytest.mark.options("Simulate=1, DriverSetup=Model:4190; BoardType:PXIe")
+    def test_perform_lcr_load_compensation(self, session):
+        session.perform_lcr_load_compensation(
+            [
+                nidcpower.LCRLoadCompensationSpot(
+                    frequency=100_000.0,
+                    reference_value_type=nidcpower.LCRReferenceValueType.IMPEDANCE,
+                    reference_value=complex(100.0, 1000.0)
+                ),
+                nidcpower.LCRLoadCompensationSpot(
+                    frequency=200_000.0,
+                    reference_value_type=nidcpower.LCRReferenceValueType.IDEAL_CAPACITANCE,
+                    reference_value=300.0e-9
+                ),
+                nidcpower.LCRLoadCompensationSpot(
+                    frequency=300_000.0,
+                    reference_value_type=nidcpower.LCRReferenceValueType.IDEAL_INDUCTANCE,
+                    reference_value=400.0e-6
+                ),
+                nidcpower.LCRLoadCompensationSpot(
+                    frequency=400_000.0,
+                    reference_value_type=nidcpower.LCRReferenceValueType.IDEAL_RESISTANCE,
+                    reference_value=200.0
+                )
+            ]
+        )
+
+
+class TestGrpc(SystemTests):
+    server_address = "localhost"
+    server_port = "31763"
+
+    def _get_grpc_server_exe(self):
+        if os.name != "nt":
+            pytest.skip("Only supported on Windows")
+        import winreg
+        try:
+            reg = winreg.ConnectRegistry(None, winreg.HKEY_LOCAL_MACHINE)
+            read64key = winreg.KEY_READ | winreg.KEY_WOW64_64KEY
+            with winreg.OpenKey(reg, r"SOFTWARE\National Instruments\Common\Installer", access=read64key) as key:
+                shared_dir, _ = winreg.QueryValueEx(key, "NISHAREDDIR64")
+        except OSError:
+            pytest.skip("NI gRPC Device Server not installed")
+        server_exe = pathlib.Path(shared_dir) / "NI gRPC Device Server" / "ni_grpc_device_server.exe"
+        if not server_exe.exists():
+            pytest.skip("NI gRPC Device Server not installed")
+        return server_exe
+
+    @pytest.fixture(scope='class')
+    def grpc_channel(self):
+        # TODO(DavidCurtiss): Remove the next 3 lines once (and the above method) the server is started automatically
+        server_exe = self._get_grpc_server_exe()
+        proc = subprocess.Popen([str(server_exe)])
+        time.sleep(3)
+        try:
+            channel = grpc.insecure_channel(f"{self.server_address}:{self.server_port}")
+            yield channel
+        finally:
+            proc.kill()
+
+    @pytest.fixture(scope='class')
+    def session_creation_kwargs(self, grpc_channel):
+        grpc_options = nidcpower.GrpcSessionOptions(grpc_channel, "")
+        return {'_grpc_options': grpc_options}
```

Comment:
Is this part intentionally different from the content of `test_system_nidmm.py`?

Reply:
To add some more context to what Tom said: In most drivers so far, for test functions that construct a Session, we have one version of the test in TestLibrary, and another in TestGrpc, where the only difference is what additional kwargs we pass to the Session constructor. Rather than duplicate such a large number of tests here, we decided to use this session_creation_kwargs fixture to supply the different kwargs per class.

For consistency, it might be a good idea to modify the system tests for all drivers to use this session_creation_kwargs style, even though the other system test files might only use it in one or two functions. But that at least doesn't need to be done in this PR, IMO.